### ISSUE TITLE
Correlation variables WITH variables

### ISSUE THREAD

**Issue by Marian-Popa** [jasp-stats/jasp-desktop#2039]
## Template for enhancement requests
 
* Enhancement: 
Option to correlate one/more variables WITH other one/more variables.
* Purpose:
It covers the situation where we are not interested in the correlation matrix, but only the correlation of certain variables WITH other variables.
SPSS has this option only through syntax, although in an old version it was included in the correlation dialog box.

Best regards,
Marian

**Comment by EJWagenmakers** [comment]
Makes sense. We'd have to see how the GUI needs to be changed to allow that.
E.J.

**Comment by Marian-Popa** [comment]
That old SPSS version had two boxes for selected variables in the correlation custom dialog, having the clause WITH between them. If the selected variables are put only in the first box, it produce a matrix correlation, otherwise it produce a correlation table between variables in the first box WITH the variabile from the second box. 
Thanks,

M.P.
